web_client/src/components/EmailDialog.vue

```diff
@@ -84,10 +84,19 @@ export default defineComponent({
       if (this.user) {
         this.cc.push(this.user.email);
       }
+
+      console.log(this.currentScan);
```

Comment:
```suggestion
      console.log(this.currentScan);
```

Reply:
Changed in [5fbde61](https://github.com/OpenImaging/miqa/pull/538/commits/5fbde613f889ef3cde2eb59ce38e845ca95c696c)